content/en/docs/reference/command-line-tools-reference/feature-gates/structured-authorization-configuration.md

```diff
@@ -0,0 +1,8 @@
+---
+title: StructuredAuthorizationConfiguration
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+Enables structured authorization configuration, which can enhances the manageability and traceability for authorization than RBAC.
```

Comment:
```suggestion
Enables structured authorization configuration, so that cluster administrators can specify more than one [authorization webhook](/docs/reference/access-authn-authz/webhook/) in the API server handler chain.
```